README.md

```diff
@@ -44,6 +44,37 @@ In order to configure some aspects of layout, like the header, edit `config/tech
 
 If you move pages around and URLs change, make sure you set up redirects from the old URLs to the new URLs.
 
+## Making changes to a diagram
+To make changes to a diagram, you'll need to edit the `drawio` files in `source/images/originals` using the draw.io desktop app.
+
+Use one drawio file per diagram.
+
+### Install draw.io desktop app
+1. Install the draw.io desktop app using Homebrew with the following command: 
+```
+brew cask install drawio
+```
+2. Alias the binary so that you can just type `draw.io` at the command line. Use the following command to do this:
```

Comment:
is there a plainer English way of saying this? Maybe 'Use the following command so you can use draw.io at the command line (this will alias the binary): '

Reply:
also would remove the word 'just' as it's not needed